doctr/models/_utils.py

```diff
@@ -69,47 +75,8 @@ def estimate_orientation(img: np.ndarray, n_ct: int = 50, ratio_threshold_for_li
     if len(angles) == 0:
         return 0  # in case no angles is found
     else:
-        return -median_low(angles)
-
-
-def get_bitmap_angle(bitmap: np.ndarray, n_ct: int = 20, std_max: float = 3.0) -> float:
```

Comment:
Why is this function not used anymore ?

Reply:
It was also not used before :sweat_smile: (there was only a lazy test for it but not used in code)

Additional it was also not working correctly with bitmaps xD 
Now one function which can handle both well